src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1175,6 +1179,13 @@ private String getBranchName(Branch branch)
         return name;
     }
 
+
+    private String getShortBranchName(Branch branch) {
+        String name = firstNonNull(branch.getName(), "");
+        Iterable<String> parts = Splitter.on("/").omitEmptyStrings().split(name);
+        return Iterables.getLast(parts);
+    }
+
```

Comment:
Hey Mark, 

That was my original intent (i even wrote a unit test to cover that) but it's a fair point - some folks name their branches with "/" in them.

While I think this satisfies the majority of branch naming conventions, setting this value as GIT_BRANCH_NAME would be incorrect. Perhaps GIT_BRANCH_SHORT_NAME ?
Any suggestions?